shuup/core/models/_products.py

```diff
@@ -132,6 +133,17 @@ def __str__(self):
 
 
 class ProductQuerySet(TranslatableQuerySet):
+
+    def _get_visible_modes(self):
+        # TODO: What's the point of this?
```

Comment:
I noticed, that when moving this to separate function the visible modes actually are "every mode ever". That's why I added the comment.